prac_05/hex_colours.py

```diff
@@ -0,0 +1,17 @@
+"""
+CP5632 Practical 5
+Convert colour name to hex_code.
+"""
+COLOUR_CODES = {"AliceBlue": "#f0f8ff", "AntiqueWhite": "#faebd7", "Aqua": "#00ffff", "Aquamarine": "#7fffd4",
+               "Azure": "#f0ffff", "Beige": "#f5f5dc", "Bisque": "#ffe4c4", "Black": "#000000",
+               "BlanchedAlmond": "#ffebcd", "Blue": "#0000ff"}
+
+colour_name = input("Enter a colour name: ")
```

Comment:
Instead of inside the loop, you can use .lower() or .title() at the user input, because by default is a string.